src/P3Scheme.jl

```diff
@@ -667,4 +767,464 @@ function D_m(p3::PSP3, q::FT, N::FT, ρ_r::FT, F_r::FT) where {FT}
     return n / q
 end
 
+"""
+    get_rain_parameters(q, N) 
+
+ - q - mass mixing ratio of rain
+ - N - number mixing ratio of rain 
+
+ returns the parameters λ and N_0 where N' = N_0 * exp(-λ  * D)
+"""
+function get_rain_parameters(q::FT, N::FT) where {FT}
```

Comment:
As discussed, could we re-use the functions (or add them if needed) in the 2-moment microphysics scheme? The same for the `get_cloud_parameters`